### ISSUE TITLE
Propose to rename the lineBroadening into Apodization

### ISSUE THREAD

**Issue by djeanner** [cheminfo/nmr-displayer#131]
Since it is not integrated in the FiltersToolBar, yet, I propose that you rename the lineBroadening function into "apodization". In this way, we will not only do exp multiplicaiton (line broadening) with it but also apply other window functions. You could add options (sin, sin2... in the menus similar to the  one for zero filling) to select the one we wish. It would be nice that if the data come from (say) Bruker data, the default apodization is the one used on the spectrometer.

**Comment by lpatiny** [comment]
Currently we have one tool bar in which we plan to put apodization and also zerofilling. What do you think ?

**Comment by djeanner** [comment]
Yes, I think that it is OK to have all window function, zero filling and (ev. lin pred...?) as a single tool bar. 

One could us it as a single filter. But your design allows (and it is nice) if we can also apply multiple window function in sequence. It is not always necessary making sense, but it may be a good way to completely control the order of the application of the operations. I suppose that a pull-down menu could be used to select the window function and (according to the choice), the parameter for the processing would be proposed (using the value from the Bruker file as default) accordingly. We can inspiration from Bruker "edp" window, the Mnova parameter window and the NMR pipe choices. For the code or the operations, I don't know if NMR pipe code is open, but it may be a way to get the window functions from. Alternative (I have not checked if they are relevant - but these are the possible source of code/inspiration) GNAT (https://www.nmr.chemistry.manchester.ac.uk/?q=node/430) , ccpm (https://www.ccpn.ac.uk/v2-software/software) - maybe you also did something?
